packages/opentelemetry-metrics/src/export/Aggregator.ts

```diff
@@ -62,7 +75,10 @@ export class MeasureExactAggregator implements Aggregator {
     this._distribution.max = Math.max(this._distribution.max, value);
   }
 
-  value(): Distribution {
-    return this._distribution;
+  toPoint(): Point {
+    return {
+      value: this._distribution,
+      timestamp: this._lastUpdateTime,
```

Comment:
last update time is always `[0,0]` unless i've missed something

Reply:
Good catch! I will fix this one and include tests.